index.html

```diff
@@ -2856,13 +2862,20 @@ <h3>Authorization</h3>
       <h2>Syntaxes</h2>
 
       <p>
-Many of the concepts in this document were introduced by example using the
-JSON syntax. This section formalizes how the data model (described in Sections
+
+The data model as described in Sections
 <a href="#core-data-model"></a>, <a href="#basic-concepts"></a>, and
-<a href="#advanced-concepts"></a>) is realized in JSON and JSON-LD. Although
-syntactic mappings are provided for these two syntaxes only, applications and
-services can use any other data representation syntax, such as XML, YAML, or
-CBOR, that is capable of expressing the data model.
+<a href="#advanced-concepts"></a> is the canonical structural
+representation of a <a>verifiable credential</a> or <a>verifiable presentation</a>. 
+All serializations are representations of that data model in a specific 
+format. This section specifies how the data model is realized in JSON-LD and plain JSON. 
+Although syntactic mappings are provided for these two syntaxes only, applications and
```

Comment:
"for these two syntaxes only" -> "only for these two syntaxes"

Reply:
This specific language was incorporated unchanged from the previous version of the specification (see the "before" side of the diff), so I'll leave it to the editors to change that as they see fit. Either is grammatically correct.

Reply:
Unfortunately, grammatical correctness does not always deliver clarity, which I think is increased by the relocated "only".  It does indeed go to the editors (named for attention raising -- @msporny @grantnoble @burnburn @dlongley).